### ISSUE TITLE
Linux Support

### ISSUE THREAD

**Issue by maverick74** [StrikeOrg/ulterius-server#65]
Hello. (First, thanks for a great project!)

I would like to ask for a Linux version.

I constantly need to access from my Linux machine to other Linux machines, as well as to other Windows machines (from Linux). Access from A Windows to a Linux could also came handy. 
This, naturally, as easy as possible!

Thanks

**Comment by keyxmakerx** [comment]
I agree, I can't use it as all my OS's are linux based. I hope one day they come out with it, as it seems to be an amazing app so long it's made well. :)
